components/suggestion/at_mention_provider/at_mention_suggestion.tsx

```diff
@@ -45,7 +62,7 @@ export default class AtMentionSuggestion extends Suggestion {
                         <span className='suggestion-list__icon suggestion-list__icon--large'>
                             <i
                                 className='icon icon-account-multiple-outline'
-                                title={title}
+                                title={title as any}
```

Comment:
Can you post the error you're getting here? I'm curious what type mismatch is happening.

Reply:
The error here is (Type 'ReactNodeArray' is not assignable to type 'string'.)

Reply:
Can we cast as `ReactNode`? I'd rather not use any if we can avoid it.